crates/swc_ecma_transforms_compat/src/es2015/regenerator/case.rs

```diff
@@ -338,8 +338,23 @@ impl CaseHandler<'_> {
             | Expr::PrivateName(..)
             | Expr::Invalid(..) => e,
 
+            // opt chain should already be transformed at this point but anyway
```

Comment:
Can we simply drop optional call here as it's es2020 feature?

Reply:
I think so, will you do it?

Reply:
Done